inventory/sample/group_vars/k8s_cluster/k8s-cluster.yml

```diff
@@ -339,3 +339,9 @@ event_ttl_duration: "1h0m0s"
 auto_renew_certificates: false
 # First Monday of each month
 # auto_renew_certificates_systemd_calendar: "Mon *-*-1,2,3,4,5,6,7 03:{{ groups['kube_control_plane'].index(inventory_hostname) }}0:00"
+
+# kubeadm patches path
+kubeadm_patches:
+  enabled: true
```

Comment:
Could we set this `false` on this inventory?
The sample inventory is almost the default option values of Kubespray because the QuickStart shows usage of the copy of that.

Reply:
Sure